OSSpecific/src/artofillusion/osspecific/MacOSPlugin.java

```diff
@@ -11,98 +11,107 @@ Changes copyright (C) 2017 by Maksim Khramov
 
 package artofillusion.osspecific;
 
-import artofillusion.*;
-import artofillusion.ui.*;
-import buoy.event.*;
-import buoy.widget.*;
-import java.awt.*;
-import java.io.*;
-import java.lang.reflect.*;
-import java.util.prefs.*;
+import artofillusion.ArtOfIllusion;
+import artofillusion.DefaultPluginImplementation;
+import artofillusion.LayoutWindow;
+import artofillusion.RecentFiles;
+import artofillusion.Scene;
+import artofillusion.SceneChangedEvent;
+import artofillusion.UndoRecord;
+import artofillusion.ViewerCanvas;
+import artofillusion.ui.EditingTool;
+import artofillusion.ui.EditingWindow;
+import artofillusion.ui.ToolPalette;
+import artofillusion.ui.Translate;
+import artofillusion.ui.UIUtilities;
+import buoy.event.CommandEvent;
+import buoy.widget.BFrame;
+import buoy.widget.BMenu;
+import buoy.widget.BMenuBar;
+import buoy.widget.BMenuItem;
+import buoy.widget.BSeparator;
+import buoy.widget.MenuWidget;
+import buoy.widget.Widget;
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.Rectangle;
+import java.io.File;
+import java.lang.reflect.InvocationTargetException;
+import java.util.prefs.PreferenceChangeEvent;
+import java.util.prefs.PreferenceChangeListener;
+import java.util.prefs.Preferences;
```

Comment:
Please do not expand package wildcard imports into these overly long lists of specific classes. As a rule of thumb, if you need more than two or three classes, import the whole package.

(This is an AOI style thing - other projects may differ)

Reply:
Resolved